sql-select/actor-forty.sql

```diff
@@ -0,0 +1,4 @@
+select "firstName",
+        "lastName"
```

Comment:
Please get _all_ columns for this one:

---

![image](https://user-images.githubusercontent.com/16143771/160526317-817baf3d-138e-4afa-8f80-b2ed73e65da9.png)